manifests/c/Canon/i-SENSYS/MF4010-MF4018/2.10/Canon.i-SENSYS.MF4010-MF4018.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Canon.i-SENSYS.MF4010-MF4018
+PackageVersion: "2.10"
+InstallerType: exe
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qn
+Installers:
+- Architecture: x64
+  InstallerUrl: https://gdlp01.c-wss.com/gds/5/0100004725/03/MF4010_MFDrivers_W64_uk_EN.exe
+  InstallerSha256: C5E0C5800DE336A886D3A00666C6FE63349F806C56D4C5760C7CCFC2914A89AA

```

```suggestion
  InstallerSha256: C5E0C5800DE336A886D3A00666C6FE63349F806C56D4C5760C7CCFC2914A89AA
    NestedInstallerType: exe
    NestedInstallerFiles: Setup.exe
      - RelativeFilePath: /
```
These are example values - please verify before confirming.